### PR TITLE
Add Contact form to the app

### PR DESCRIPTION
**Pull request by JohnFTitor**
# Hi there 👋🏻 

Thanks for taking the time to review the project 💯 🐱 

### Changes implemented in "contact-form" branch 📱  

✅ Follow style guidelines
✅ Created a contact form set to my Formspree account
✅ Styled elements to meet UX design, following the template provided
✅ Added a footer with social media links 
✅ We work together with the pair programming technique 
 
## Final notes

Is expected that this project meets the requirements. Please let me know if anything is left to be done or can be improved. Again, thanks for the review.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,200 +1,259 @@
...
+            </li>
+        </ul>
+    </section>
+    <section class="container">
+        <div class="content contact">
+            <p> I'm always interested in hearing about new projects, so if you'd like to chat please get in touch. </p>
+            <form class="contact" action="https://formspree.io/f/xqknzgnq" method="post">
+                <input name="user-name" type="text" id="name" placeholder="Full name" maxlength="30" required>
+                <input name="user-email" type="email" id="email" placeholder="Email address" required>
+                <textarea name="comment" maxlength="500">Enter text here</textarea>
```
Comment (VanessaAoki) [inline index.html]: - [ ] Great job setting up formspree correctly 👏  you should consider adding "required" to the `<textarea>`, I was able to submit the form with a blank field.

File: index.html [hunk 2]
```diff
@@ -1,200 +1,259 @@
...
-                            <li class="textPlaceholder tag"><a rel="noopener noreferrer" href="#" aria-label=""> html </a></li>
-                            <li class="textPlaceholder tag"><a rel="noopener noreferrer" href="#" aria-label=""> bootstrap </a></li>
-                            <li class="textPlaceholder tag"><a rel="noopener noreferrer" href="#" aria-label=""> Ruby </a></li>
-                        </ul>
-                    </div>
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale= 1.0">
+    <title> My portfolio </title>
```
Comment (VanessaAoki) [inline index.html]: - [ ] [OPTIONAL]: To keep your project even more professional, you should consider adding a more descriptive title, what about having "your name's portfolio", for instance?

**Review by VanessaAoki — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED :yellow_circle: 
Hi @JohnFTitor  and @vanillainanutshell , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

### Project highlights:
- [x] All linter checks are passing :heavy_check_mark: 
- [x] Your README file looks very professional and the PR has a descriptive title/summary of the implemented changes :memo: 
- [x] Descriptive commits and correct use of the GitHub Flow :fire: 
- [x] Used flexbox to style the form elements 🎨  

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

#### You should also consider:
- [ ] To keep your repo professional, you should consider editing the README author's section, adding your social media!

so close

Cheers and Happy coding! 👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by rotimiazeez — APPROVED** [review]
# Hi Team😉, 

### Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 👏
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**